spec/requests/event_steps_controller_spec.rb

```diff
@@ -179,11 +179,25 @@ def perform_request
   end
 
   describe "#completed" do
+    let(:path) { completed_event_steps_path(readable_event_id) }
```

Comment:
Don't see aspecific reason why this is broken out into a `let`, is this a left-over from aborted changes below?

Reply:
Ah yeah I started doing a refactor of the specs above to consolidate the common setup but then decided against it. I've removed the let and fixed up the commit if you could re-tick it when you have a minute.